### ISSUE TITLE
PivotTable case sensitivity problem with RowLabels

### ISSUE THREAD

**Issue by chris-denning** [ClosedXML/ClosedXML#123]
I've found a bug in the PivotTable implementation that depends on the data rather than the code.

I use ClosedXML to produce an export of purchase transaction information, and add a pivot table with "Category" and "Item" as the row labels, and "Total price" as the value.

Usually this works fine, but as the dataset gets bigger I have discovered that if there are any Item labels that differ only in capitalisation, it results in an invalid spreadsheet.

In this case Excel 2016 gives the error "We found a problem with some content ..." and the recovery log says:

```XML
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<recoveryLog xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main"><logFileName>error024240_05.xml</logFileName><summary>Errors were detected in file 'C:\Users\chris\Downloads\order-lines (1).xlsx'</summary><removedFeatures><removedFeature>Removed Feature: PivotTable report from /xl/pivotTables/pivotTable.xml part (PivotTable view)</removedFeature></removedFeatures><removedRecords><removedRecord>Removed Records: Workbook properties from /xl/workbook.xml part (Workbook)</removedRecord></removedRecords></recoveryLog>
```

Problematic data example:

Category | Item | Price
-------- | ---- | -----
Registration | Early registration | 1
Registration | EARLY registration | 1
Registration | Late registration | 2

If I change "EARLY" to "Early" it works okay.

Here's a code snippet showing how I add the pivot table, but I don't think it's a problem with my code

```C#
      // Add pivot table
      var dataRange = sheet.RangeUsed();

      var pivotTableSheet = workbook.Worksheets.Add("PivotTable");

      var pt = pivotTableSheet.PivotTables.AddNew("PivotTable", pivotTableSheet.Cell(1, 1), dataRange);

      pt.RowLabels.Add("Category");
      pt.RowLabels.Add("Item");

      //pt.ColumnLabels.Add("Item");

      pt.Values.Add("Total price");
```

**Comment by chris-denning** [comment]
Oh, and I'm assuming that ColumnLabels will be affected as well, although I haven't tested.

**Comment by igitur** [comment]
Fixed in #128 . Thanks for the very clear steps to reproduce.
